clients/src/main/resources/common/message/CreateTopicsRequest.json

```diff
@@ -48,7 +48,7 @@
         { "name": "BrokerIds", "type": "[]int32", "versions": "0+", "entityType": "brokerId",
           "about": "The brokers to place the partition on." }
       ]},
-      { "name": "Configs", "type": "[]CreateableTopicConfig", "versions": "0+",
+      { "name": "Configs", "type": "[]CreatableTopicConfig", "versions": "0+",
```

Comment:
Please fix all references, otherwise, the build can't pass

Reply:
Thanks for pointing out the error. I've fixed all references, and the code now can builds successfully.